playbooks/common/openshift-cluster/upgrades/v3_11/upgrade_control_plane.yml

```diff
@@ -66,6 +66,14 @@
 # TODO: need to verify settings about the bootstrap configs
 # 1. Does network policy match the master config
 
+- name: Ensure metrics-server is installed before upgrading the controller-manager
+  hosts: oo_first_master
+  roles:
+  - role: metrics_server
+    # if a user already had metrics installed, they already had a working HPA, so don't break them
+    when: openshift_metrics_server_install | bool
```

Comment:
Needs `default(true)` to prevent from failing when this var is not defined

Reply:
There's a default added in the role.  Shouldn't that mean it's always defined?